upsilonconf/io.py

```diff
@@ -1,7 +1,7 @@
 import json
 from argparse import ArgumentParser, Namespace
 from pathlib import Path
-from typing import Union, Any, Sequence, Mapping, Callable, overload, Tuple
+from typing import Union, Any, Sequence, Mapping, Callable, overload, Tuple, Dict
```

Comment:
Is this necessary? Shouldn't everything also work with (the more general) `Mapping` type?

Reply:
Honestly, I don't know. I was never a big fan of typing, so thought, I'll use `Dict` for the return value of the `__replace_in_keys()` functions. What do you propose?

Reply:
`Mapping` is a bit more general than `Dict`, so I would prefer to use `Mapping` instead of `Dict`

Reply:
Done in https://github.com/hoedt/upsilonconf/pull/3/commits/b1d08be48cec06b97e841aea8c87fd7ab586d71c.